cli/module_registry.go

```diff
@@ -569,8 +569,11 @@ func validateModuleFile(client *viamClient, c *cli.Context, moduleID moduleID, t
 		}
 		path := header.Name
 
-		// if path == entrypoint, we have found the right file
-		if filepath.Clean(path) == filepath.Clean(entrypoint) {
+		// if path == entrypoint, we have found the right file. On Windows, allow for an optional
+		// ".exe" at the end.
```

Comment:
optionally add explanation here that windows auto resolves `entrypoint: bin` to `bin.exe`